views/index.ejs

```diff
@@ -3,6 +3,19 @@
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 
+        <!-- Buttons -->
+        <div class="topnav">
+          <button id = "login" > Login </button>
+          <button id = "logout" >Logout </button>
```

Comment:
It's actually meant to just be a single button that will change to `Logout` when you are logged in, which isn't working quite yet. For now, as the issue mentions, please have this button say "Login" and link to `/login`. It can also be a `<a>` tag if that makes it easier but styled like a button.